servo-tidy.toml

```diff
@@ -46,3 +46,8 @@ directories = [
   "./target",
   "./ports/cef",
 ]
+
+# Directories that are checked for correct file types
+[check_dirs_for_exts]
```

Comment:
No, it may not be the best, I just thought we could keep the table names as precise as possible :)